src/main/java/com/google/api/codegen/transformer/py/PythonGapicSurfaceTransformer.java

```diff
@@ -109,7 +109,8 @@ public PythonGapicSurfaceTransformer(
   }
 
   @Override
-  public List<ViewModel> transform(Model model, GapicProductConfig productConfig) {
+  public List<ViewModel> transform(ApiModel apiModel, GapicProductConfig productConfig) {
+    Model model = ((ProtoApiModel) apiModel).getProtoModel();
```

Comment:
Add TODOs for casts like this that we will have to address later

Reply:
Done.